paper_to_git/bin/paper-git.py

```diff
@@ -1,4 +1,4 @@
-"""The paper_git command dispatcher"""
+"""The paper-git command dispatcher"""
```

Comment:
I think command module should remain same as `paper_git` as python doesn't allow `-` in the module names and hence they can't be imported.